statistics-dashboard-service/src/main/java/eu/europeana/statistics/dashboard/service/StatisticsService.java

```diff
@@ -183,6 +222,9 @@ private StatisticsQuery prepareFilteringQuery(StatisticsFilteringRequest statist
   }
 
   private double calculatePercentage(double totalCount, double count) {
+    if (totalCount <= 0) {
```

Comment:
This is great, but I would intrigue you to try to make this whole method a one liner return :)

Reply:
I did it! :)